code/__DEFINES/projectiles.dm

```diff
@@ -36,6 +36,8 @@
 #define CALIBER_A762 "a762"
 /// The caliber used by shotguns.
 #define CALIBER_SHOTGUN "shotgun"
+/// The caliber used by improvised shotgun shells.
+#define CALIBER_IMPROV_SHOTGUN "improvised_shotgun"
```

Comment:
You could make this just a general CALIBER_IMPROVISED. No reason not to open up the potential for more unique ammo down the line. I get the feeling a non-standard gun just accepting 'whatever bullets that fit in the weird tubes we make' sounds kinda funny.

Reply:
I dunno, that seems a bit excessive